tests/benchmarks/class-internal.rkt.out

```diff
@@ -1345,10 +1357,12 @@
                            [(local-field ...) (definify (append field-names private-field-names))]
                            [(local-field-localized ...)
                             (map lookup-localize (append field-names private-field-names))]
-                           [(local-field-pos ...)
-                            (let loop ([pos 0]
-                                       [l (append field-names private-field-names)])
-                              (if (null? l) null (cons pos (loop (add1 pos) (cdr l)))))]
+                           [(local-field-pos ...) (let loop ([pos 0]
```

Comment:
@jackfirth this is what I told you earlier... Probably there should be a newline here. But let's not worry about it in this PR.